aip/general/0193.md

```diff
@@ -62,11 +62,120 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
+### Error messages
 
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+For each error, the service **must** populate the `message` field on
+[`google.rpc.Status`][Status]. This error message,
+
+- is a developer-facing, human-readable "debug message"
+- is presented in the service's native language
+- both explains the error and offers an actionable resolution to it
+  ([citation](https://cloud.google.com/apis/design/errors#error_model))
+
+**Note:** Sometimes a service will elect to always present
+`Status.message` in English rather than the application's native
+language so that messages are easily searchable in common knowledge
+bases, such as StackOverflow&trade;.
+
+When introducing a new error, a failure scenario that did not previously
+occur for the service, the payload **must** include `ErrorInfo` and any
+variables found in dynamic segments of the error message **must** be
+present in `ErrorInfo.metadata`. See, "[Dynamic
+variables](#dynamic-variables)".
+
+#### Changing error messages
+
+If your service's API design governing body decides that altering
```

Comment:
these are Google AIP's, so we should know whether we govern this or not, correct? and state that clearly here.

Reply:
Some PAs have their own governance.

Reply:
I think this should be addressed - the guidance is vague and for someone arriving at this page, unclear as to how to figure out whether this is backwards incompatible or not.

Can it be explicitly stated that the change is allowed at minimum? the *must not* in here should be added by the PA in their own guidance then.

Reply:
I changed it so it explicitly states that `Status.message` **may** change if not disallowed. Is this what you're after?

Reply:
I'm more not a fan of the "governing bodies" which is vague and leaves a reader unable to figure out what to do next. 

If I was new to Google and I read this sentence, what would I do next? how would I find out if there was some other governing body?

I don't think we really need to discuss that this could or could not be a backwards incompatible change, unless we're going to say something authoritative like *should* or *should not*.